### PR TITLE
Feature/ps ip

### PR DESCRIPTION
**Pull request by EldritchJS**
## Related Issues and Dependencies
NA
…

## This introduces a breaking change

- [ ] Yes
- [X ] No

## This Pull Request implements

Adds s2i and Thoth niceness

## Description

First iteration of ps-ip that *should* be buildable

### CODE REVIEW

File: thoth/ps-ip/__init__.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python3
-# project template
-# Copyright(C) 2021 Red Hat, Inc.
+# ps-nlp
```
Comment (codificat) [inline thoth/ps-ip/__init__.py:2]:
```suggestion
# ps-ip
```

File: .prow.yaml [hunk 2]
```diff
@@ -6,7 +6,7 @@ presubmits:
     context: aicoe-ci/prow/pre-commit
     spec:
       containers:
-        - image: quay.io/thoth-station/thoth-precommit-py38:v0.12.9
+        - image: quay.io/thoth-station/thoth-precommit-py38:v0.12.7
```
Comment (pacospace) [inline .prow.yaml:9]: v0.13.0

File: overlays/ps-ip-ifd/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+LABEL name="ps-ip-ifd:latest" \
+      summary="Image Processing Source-to-Image for Python 3.8 applications." \
+      description="Notebook image based on Source-to-Image.These images can be used in OpenDatahub JupterHub." \
+      io.k8s.description="Notebook image based on Source-to-Image.These images can be used in OpenDatahub JupterHub." \
+      io.k8s.display-name="Image Processing Notebook Python 3.8-ubi8 S2I" \
+      io.openshift.expose-services="8888:http" \
+      io.openshift.tags="python,python38" \
+      authoritative-source-url="https://quay.io/thoth-station/ps-ip-ifd" \
+      io.openshift.s2i.build.commit.ref="master" \
+      io.openshift.s2i.build.source-location="https://github/thoth-station/ps-ip-ocr" \
```
Comment (pacospace) [inline overlays/ps-ip-ifd/Dockerfile:11]: `ps-ip-ifd`

**Pull request merged**